### DISCUSSION TITLE
How can I use it in shell script?

### DISCUSSION THREAD

**Discussion by andersonid** [balta2ar/brotab#83] [Q&A]
Hello everyone!
I'm trying to use the brotab command inside a shell script with no luck! :-1: 
This line of the code works like a charm on terminal. But I cant get work in script.

`
#!/bin/bash`
`bt navigate $(bt new $(bt list | grep -i gmail | grep -o '^\S*' | cut -c1-1)) "https://calendar.google.com"
`

What's wrong with shell script code if is the same piece of sh#$%! that works on terminal?

**Comment by balta2ar** [comment]
How do you run the script? Your PATH is likely different when you call it. Try this in terminal:
```
$ which bt
/home/user/.local/bin/bt
```
For example. And then put that path into a var in the script:
```
BT="/home/user/.local/bin/bt"
$BT list
# blah blah
```
